addons/stock/models/stock_location.py

```diff
@@ -174,6 +174,11 @@ def should_bypass_reservation(self):
         self.ensure_one()
         return self.usage in ('supplier', 'customer', 'inventory', 'production') or self.scrap_location
 
+    @api.constrains('location_id', 'location_id')
```

Comment:
`@api.constrains('company_id', 'location_id')`, right? 🤔

Reply:
@mreficent oups, you're right ! thanks ;)